pkgs/development/python-modules/vispy/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, buildPythonPackage, substituteAll, stdenv,
+  fetchPypi, numpy, cython, freetype-py, fontconfig, libGL,
+  setuptools, setuptools_scm, setuptools-scm-git-archive,
+  pyqt4
+  }:
+
+buildPythonPackage rec {
+  pname = "vispy";
+  version = "0.6.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "07sb4qww6mgzm66qsrr3pd66yz39r6jj4ibb3qmfg1kwnxs6ayv2";
+  };
+
+  patches = [
+    (substituteAll {
+      src = ./library-paths.patch;
+      fontconfig = "${fontconfig.lib}/lib/libfontconfig${stdenv.hostPlatform.extensions.sharedLibrary}";
+      gl = "${libGL.out}/lib/libGL${stdenv.hostPlatform.extensions.sharedLibrary}";
+    })
+  ];
+
+  buildInputs = [
+    cython setuptools setuptools_scm setuptools-scm-git-archive pyqt4
```

Comment:
aside from pyqt4 the others likely belong in `nativeBuildInputs`. `setuptools` should not be here, unless it's needed at runtime in which case it should be in `propagatedBuildInputs`

Reply:
updated as requested, `setuptools` and `pyqt4` removed (`pyqt4` was part of a failed effort to make tests work.)